tests/test_helm.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+CURRENT_DIR=$(dirname $(realpath $0))
+SEEKRET_IS_RUNNING=0
+TEST_ENV_IS_RUNNING=0
+NAMESPACE=${NAMESPACE:-"cdtests"}
+
+cleanup() {
+  if [ $TEST_ENV_IS_RUNNING -eq 1 ]; then
+		run kubectl delete deployment hello-node -n $NAMESPACE
+	fi
+	if [ $SEEKRET_IS_RUNNING -eq 1 ]; then
+		run helm uninstall seekret -n $NAMESPACE
+	fi
+
+  if [ $1 -ne 0 ]; then
+    echo "Failed."
+  else
+    echo "Success"
+  fi
+	exit $1
+}
+
+cleanupOnError()
+{
+  if [ $1 -ne 0 ]; then
+    cleanup $1
+  fi
+}
+
+run()
+{
+  if [ $OUTPUT -eq 0 ]; then
+    $@ > /dev/null 2>&1
+  else
+    $@
+  fi
+
+}
+
+if [ -z $1 ] || [ -z $2 ] || [ -z $3 ]; then
+	echo "usage: $(basename $0) <access key> <secret key> <bucket name> [-v optional]"
+	exit 1
+fi
+
+OUTPUT=0
+
+if [ ! -z $4 ] && [ "$4" = "-v" ]; then
+  OUTPUT=1
+fi
+
+ACCESS_KEY=$1
+SECRET_ACCESS_KEY=$2
+BUCKET_NAME=$3
+echo "Creating seekret's helm sniffer setup"
+run helm install seekret --namespace $NAMESPACE --create-namespace ${CURRENT_DIR}/../helm/seekret/ --set s3.accessKey=${ACCESS_KEY} --set s3.secretKey=${SECRET_ACCESS_KEY} --set s3.bucketName=${BUCKET_NAME}
```

Comment:
I think that we dont need a namespace anymore since we have no resource in that namespace

Reply:
for the scenario you will run the test locally, I don't want to pollute any locally configured environment

Reply:
Well currently we cant instal our sniffer twice no matter the namespace, but I believe we will fix that in the future so lets leave this